### PR TITLE
new pre-written payments

### LINKED ISSUE #142: Add previous transactions for new users

### PR DESCRIPTION
**Pull request by adendiamond**

### CODE REVIEW

**Review by Barradism — APPROVED** [review]
👌🏻🫡

File: learn-to-bank/app/service/userService.ts [hunk 1]
```diff
@@ -93,6 +93,43 @@ export async function createUser(context: any, uid: string, email: string, first
...
+                description: 'Settle credit payment',
+                frequency: JSON.stringify(generateRandomFrequencyObject()),
+                starts_on: new Date(),
+                ends_on: null,
+            }
+        });
+
+        // Five previous payments
+        for (let i = 0; i < 5; i++) {
+            const amt = Math.floor(Math.random() * 10000)
```
Comment (arjadas) [inline learn-to-bank/app/service/userService.ts:114]: are the transactions random for every new user or every time the page is accessed?

**Review by arjadas — APPROVED** [review]
looks good.

File: learn-to-bank/app/service/userService.ts (hunk 1 above)
Comment (adendiamond) [reply learn-to-bank/app/service/userService.ts:114]: they're created on sign-up.

**Review by TharathepC — APPROVED** [review]
Great!

**Pull request merged**